docs/spec/special-types.rst

```diff
@@ -175,7 +175,10 @@ Note that it is legal to use a union of classes as the parameter for
       user = new_user(user_class)
       ...
 
-However the actual argument passed in at runtime must still be a
+``type[]`` distributes over unions:
+``type[A | B]`` is consistent with ``type[A] | type[B]``.
```

Comment:
I think the right term to use is "equivalent" (https://typing.readthedocs.io/en/latest/spec/glossary.html#term-equivalent).

Reply:
If I'm understanding the terms correctly, `equivalent` applies only to fully-static types. I think we want to accommodate types that include gradual forms here, no? After all, `type[int] | type[list[Any]]` describes the same set of values as `type[int | list[Any]]`.

Reply:
No, equivalent also applies to types that are not fully static; see the linked definition. Fully static types are equivalent if they cover the same set of values; types that are not fully static are equivalent if the set of types they can materialize to is the same.

Reply:
Ah yes, I was thrown off by [this table](https://typing.readthedocs.io/en/latest/spec/concepts.html#summary-of-type-relations). I should have read the paragraph below the table.